project/CommonSettings.scala

```diff
@@ -116,4 +118,6 @@ object CommonSettings {
   ) ++ testSettings
 
   lazy val prodSettings: Seq[Setting[_]] = settings
+
+  lazy val binariesPath = Paths.get(Properties.userHome, ".bitcoin-s", "binaries")
```

Comment:
Any reason not to read the conf file and go to the correct `baseDataDir`? What if someone is running from a different datadir?

Reply:
You are talking about running the application, where as this is for running tests. You don't need an `AppConfig` to run `bitcoindRpcTest` or `eclairTest`.